src/components/organisms/SideBar/SideBar.tsx

```diff
@@ -44,7 +44,11 @@ const MenuItems: FC<MenuItemsProps> = ({
       {map(
         menuItems,
         ({ children, path, Icon, isInterTitle, label, privileges }) => {
-          const fullPath = parentPath ? `${parentPath}/${path}` : `/${path}`
+          const fullPath = parentPath
+            ? path
```

Comment:
**[non-blocker]**

Would suggest avoiding ternary operators, if they have to be nested. The readability can get bad really quickly.